pkg/gds/admin.go

```diff
@@ -1006,6 +1007,133 @@ func (s *Admin) Resend(c *gin.Context) {
 	c.JSON(http.StatusOK, out)
 }
 
+// ReviewTimeline returns a list of time series records containing registration state counts by week.
+func (s *Admin) ReviewTimeline(c *gin.Context) {
+	// Go needs this constant to determine the time format
+	const timeFormat = "2006-01-02"
+	var (
+		err          error
+		in           *admin.ReviewTimelineParams
+		out          *admin.ReviewTimelineReply
+		startTime    time.Time
+		endTime      time.Time
+		earliestTime time.Time
+		weekTime     time.Time
+		numWeeks     int
+		vaspCounts   []map[string]bool
+	)
+
+	// Get request parameters
+	in = new(admin.ReviewTimelineParams)
+	if err = c.ShouldBindQuery(&in); err != nil {
+		log.Warn().Err(err).Msg("could not bind request with query params")
+		c.JSON(http.StatusBadRequest, admin.ErrorResponse(err))
+		return
+	}
+
+	// Default value for start date
+	earliestTime = time.Date(2020, time.January, 1, 0, 0, 0, 0, time.Local)
```

Comment:
Times are stored in the database as UTC - perhaps it would be better to use `time.UTC` for the tzinfo in this date?

Reply:
Agreed, that makes more sense.